wscript

```diff
@@ -211,7 +211,8 @@ def rebuildSubscribableWebObjectsWrapper(bld):
 
 def build(bld):
     """Primary WAF entry point."""
-    bld.post_mode = Build.POST_LAZY
+    bld.post_mode = Build.POST_AT_ONCE
+    bld.jobs = 4
```

Comment:
What's the difference between POST_LAZY and POST_AT_ONCE? 
I don't think you want to restrict bld.jobs to 4 (assuming it acts like the -j option of make to controll how many concurrent processes are allowed). During local builds, ./waf uses as many processes as my cores by default, so I think you may want to rever this change as well.